### ISSUE TITLE
Emoji search

### ISSUE THREAD

**Issue by rugk** [threema-ch/threema-web#789]
It would be great, if one could search for the emoji by it's name (and special keywords etc.; as defined per [CL;DR](http://cldr.unicode.org/) – yes, they have it, even translated/localized, but don't ask me where).

In the meantime, I've found a [different solution already](https://addons.mozilla.org/firefox/addon/awesome-emoji-picker/), but well… :wink: 

I know some competitioners (:wink:) do have this feature in their web UI.

**Comment by lgrahl** [comment]
Duplicate of #433.

**Comment by rugk** [comment]
No, no, I want a "search box" that I can type into. So you can open the emoji picker as it currently is, and then type in a keyword, and it shows you all possible emojis that match that one…

Many chat services (Mattermost, etc.) both have this "type to input emoji" feature and a _seperate_ emoji picker, where you can also search.

The `:something` may be an alternative solution to the same use case, but is not actually the same… :thinking: And I guess there are some advantages of a real picker search, e.g. that users not knowing the `:` syntax get it, and that you can see more (all) suggestions/results instead of just a limited set.

**Comment by dojo90** [comment]
If you use Windows 10, try:
<kbd>⊞ Win</kbd>+<kbd>.</kbd>

**Comment by annabekka** [comment]
A search box for emojis within threema web would still be great

**Comment by rugk** [comment]
> If you use Windows 10, try:

…or, if you use Firefox, [use a Firefox add-on](https://addons.mozilla.org/firefox/addon/awesome-emoji-picker/) lol, sure. That is my personal workaround (obviously :wink:), but the included version would be better.

**Comment by joelfischerr** [comment]
On macOS you can try Rocket by Matthew Palmer: https://matthewpalmer.net/rocket/

**Comment by reitzig** [comment]
If we're supposed to use external tools, Threema can drop the emoji list altogether.

Having a list with thousands of entries without a way to search seems ... pointless. Does anybody actually scroll through that behemoth? :astonished:

**Comment by foreachthing** [comment]
That's one thing I miss over whatsapp ... ability to search for emoji by keyword; I.e. "wink", "flag", etc.
Also, that it converts ;-) to 😉 , the correct emoji. But that's a different "issue".

**Comment by threema-danilo** [comment]
For the people that aren't aware yet: If you hover over an emoji in the picker, you can see its shortcode. That shortcode is already converted to an emoji when typing, there's simply no search or autocompletion. But if you type `:tada:` that will be converted to 🎉.

**Comment by foreachthing** [comment]
@threema-danilo, sure. But that's only true for threema-web (sorry, can't post an issue for threema-Android), on my phone it's a different story. Nothing is converted, no emoji-markups are possible.
